modules/eks/variables.tf

```diff
@@ -199,3 +199,41 @@ variable "tags" {
   description = "Deployment tags."
   default     = {}
 }
+
+variable "privatelink" {
```

Comment:
this var should be replicated in `modules/eks/submodules/privatelink/variables.tf` see how `eks` var is passed

Reply:
done